linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_forward_files/rule.yml

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
```

Comment:
Please add a `prodtype`.

Reply:
:+1: added in [478f10d](https://github.com/ComplianceAsCode/content/pull/9990/commits/478f10d91237b340025fe3ca5be521f0347e85d2)